civihr_employee_portal/views/templates/views-view--hr-resource-types-list--page.tpl.php

```diff
@@ -94,4 +94,3 @@
 <?php endif; ?>
 
   </div><?php /* class view */ ?>
-
```

Comment:
Why was the line at the end of the file removed? I thought this was a good practice.

Reply:
Yeah it was removed by mistake, thanks for noticing.